src/python3_pip_skeleton/__main__.py

```diff
@@ -164,10 +166,86 @@ def new(args):
 
 
 cfg_issue = """Missing parameter in setup.cfg. Expected format:
-[metadata]
-name = example
-author = Firstname Lastname
-author_email = email@address.com"""
+    [metadata]
+    name = example
+    author = Firstname Lastname
+    author_email = email@address.com
+
+    ------- pyproject.toml
+    [[project.authors]]
+    name = "Firstname Lastname"
+    email = "email@address.com"
+"""
+
+
+def obtain_author_name_email(path: Path) -> tuple:
```

Comment:
Shouldn't this function be used in `existing()`? What about `new()`?

Reply:
I've added the function to `existing()`. It wont be used in `new()` however I've added a function to get the git details which both `new()` and `existing()` will call.